portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,10 +48,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<String> comments = new ArrayList<String>();
+    HashMap<String, ArrayList<String>> comments = new HashMap<String, ArrayList<String>>();
```

Comment:
Would _new HashMap<>();_  be sufficient?

Reply:
yep, updated

Reply:
The code still looks the same.

Reply:
<img width="575" alt="Screen Shot 2020-12-21 at 1 07 18 pm" src="https://user-images.githubusercontent.com/48550736/102731984-7d812f00-438d-11eb-8923-14f24bb29696.png">
I have no idea why. This is my code in the shell editor. I have tried to push the updates but it says that everything is up to date.

Reply:
I will ask tech support now

Reply:
ok so when you click on view file you need to make sure the right branch/latest commit is selected to see all the changes